### PR TITLE
4.6 After installation infra and audit index pattern not available in Kibana

### PR DESCRIPTION
**Pull request by mburke5678**
Incorporating changes in https://github.com/openshift/openshift-docs/pull/24579 to the 4.6 docs.

### CODE REVIEW

**Comment by mburke5678** [comment]
Update this link after https://github.com/openshift/openshift-docs/pull/24029 is merged: https://github.com/openshift/openshift-docs/pull/25456/files#diff-b72f068ee90a4f7fb23e14c47886ffb8R41

**Comment by mburke5678** [comment]
@anpingli Can you PTAL at this PR? Small changes to the 4.6 doc in order to add https://github.com/openshift/openshift-docs/pull/24579. 
* Removed the step to add ` clusterlogging.openshift.io/logforwardingtechpreview: enabled` to enable Log Forwarding.
* Changes to CR (`endpoint` to `url` and `inpurSource` to `inputRefs`.
* Changed a note to say that user `kubeadmin` does have proper permissions to list *infra* and *audit* indices in Kibana.

**Review by anpingli — COMMENTED** [review]
@mburke5678  Which type of LogForwarding is this for? TP or GA.   The url and inputRefs belong to GA LogForward,  and the kind is ClusterLogForwarder.

**Comment by mburke5678** [comment]
@anpingli This is for GA log forwarding.

File: modules/cluster-logging-elasticsearch-audit.adoc [hunk 1]
```diff
@@ -18,67 +18,35 @@ The internal {product-title} Elasticsearch log store does not provide secure sto
...
-<1> Enables and disables the Log Forwarding API. Set to `enabled` to use log forwarding. 
-<2> The `spec.collection` block must be defined to use Fluentd in the Cluster Logging CR.
-
 . Create a Log Forwarding CR YAML file or edit your existing CR:
 +
 * Create a CR to send all log types to the internal Elasticsearch instance. You can use the following example without making any changes:
 +
 [source,yaml]
 ----
 apiVersion: logging.openshift.io/v1alpha1
```
Comment (anpingli) [inline modules/cluster-logging-elasticsearch-audit.adoc:27]: apiVersion: logging.openshift.io/v1

Comment (anpingli) [reply modules/cluster-logging-elasticsearch-audit.adoc:27]: @mburke5678  You needn't to defin the outputs sections in ClusterLogForwarder. There is a predefined keyword 'default'
apiVersion: logging.openshift.io/v1
kind: ClusterLogForwarder
metadata:
  name: instance
  namespace: openshift-logging
spec:
  pipelines:
    - name: test-infra
      inputRefs:
      - infrastructure
      - application
      - audit
      outputRefs:
      - default

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**